isce2_topsapp/localize_mask.py

```diff
@@ -19,17 +19,27 @@ def download_water_mask(
         np.ceil(extent_buffered[3]),
     ]
 
-    if water_name == "SWDB":
+    if water_name == "SWBD":
         # Download SRTM-SWDB water mask
-        mask_filename = download_wbd(
-            extent_buffered[1],
-            extent_buffered[3],
-            extent_buffered[0],
-            extent_buffered[2],
-        )
+        # Water mask dataset extent
+        # Latitude S55 - N60
+
+        lats = [extent_buffered[1], extent_buffered[3]]
+        if (np.abs(lats) < 59.9).all():
+            mask_filename = download_wbd(
+                extent_buffered[1],
+                extent_buffered[3],
+                extent_buffered[0],
+                extent_buffered[2],
+            )
+            mask_filename = str(output_dir / mask_filename)
+        else:
+            print('Request out of SWBD coverage ',
+                  'Skip downloading water mask!!')
+            mask_filename = ''
```

Comment:
Can you please clarify why you don't raise an error? If there is no water mask available, will ionosphere process run to completion yielding an all zero layer? Or will something else occur?

I think we may want to raise an error in this case because ionosphere should not be computed right? However, I do like the idea that we can create generic products beyond a water mask coverage area. What are your thoughts.

Generally, operator (person submitting jobs on hyp3) will avoid these higher/lower latitudes.

Reply:
There is no error as ionosphere computation will go on without using water mask - it will return to default processing with minor modifications. 
1. iono will result in all zero only if there is no land/or data to compute iono
2. we do not need water mask to do iono computation, we use it to improve iono near coastline where mask is available. To be more specific, we are masking water out before unwrapping lower and upper band
3. I think we can still run high latitudes with respect to water mask which is optional, but we do need dem. I would suggest to make sure that DEM coverage is documented is Warning is raised if it is out of its scope.